cmd/template/cluster/flag.go

```diff
@@ -112,8 +112,8 @@ func (f *flag) Validate() error {
 		// Validate Master AZs.
 		switch f.Provider {
 		case key.ProviderAWS:
-			if len(f.MasterAZ) != 0 && len(f.MasterAZ) != 1 && len(f.MasterAZ) != 3 {
-				return microerror.Maskf(invalidFlagError, "--%s must be set to either one or three availability zone names", flagMasterAZ)
+			if len(f.MasterAZ) < 1 && len(f.MasterAZ) > 3 {
```

Comment:
Not sure if the length can be both less than 1 and more than 3 at the same time 😬 

```suggestion
			if len(f.MasterAZ) < 1 || len(f.MasterAZ) > 3 {
```